ui/js/controllers/jobs.js

```diff
@@ -71,6 +71,10 @@ treeherderApp.controller('JobsCtrl', [
             }
         };
 
+        if($location.search().revision === 'undefined'){
```

Comment:
So for best practices style wise here, the **if** ideally has spaces on either side of the conditional
```
if (condition) {
```